src/components/pages/home/introduction.tsx

```diff
@@ -39,13 +63,24 @@ const Introduction: React.FC = () => {
     },
   ]
 
+  const onVideoEnd = useCallback(() => {
+    if (videoRef.current) {
+      videoRef.current.showPreview()
+    }
+  }, [videoRef])
+
   return (
     <Wave
-      style={bgImgStyle}
+      style={{ zIndex: 1 }}
       wave="white"
       spacing="sm"
       waveClassNames="bg-contain lg:bg-auto xl:bg-contain"
     >
+      <div
+        className="absolute left-0 right-0 lottie-player"
+        ref={setBgRef}
+        style={{ zIndex: '-1' }}
```

Comment:
Could we extend the z-index in tailwind to include a negative index?

https://tailwindcss.com/docs/z-index#negative-values